express-notes-api/index.js

```diff
@@ -0,0 +1,85 @@
+const express = require('express');
+const app = express();
+const PORT = 3000;
+
+let nextId = 1;
+const notes = {};
```

Comment:
The `data` should be coming from the `data.json` file here. You should use `require` to get the `data`